pkg/controllers/subnetipreservation/subnetipreservation_controller.go

```diff
@@ -138,14 +124,6 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	}()
 	r.StatusUpdater.IncreaseSyncTotal()
 
-	if !r.IPReservationService.Supported {
```

Comment:
Why this patch removes the logic to detect whether IPReservationService is supported? Is it ok to continue the reconciling for the not supported case?

Reply:
Add NSXVersion as discussed